### ISSUE TITLE
Two-way binding

### ISSUE THREAD

**Issue by stephenlautier** [erikringsmuth/nex-js#1]
Hi,

I'm currently using this in a small project since its quite simple the way it works along with the plugins used in http://erikringsmuth.github.io/nex-js/#/examples e.g. requirejs etc... and I quite liked it  :).

However there's a missing feature (or I'm missing it) which I'd like to have which is Two-way binding, any suggestions how i can have two waybinding with this? or any suggestion which plugin goes well with it?

Basically what I need is this:
model -> bound to a input text -> which updates the model back -> render it again in a different form 

Small example such as http://emberjs.com/ - AUTO-UPDATING HANDLEBARS TEMPLATES. However my scenario is much more complex e.g. not simply 1 textbox, the UI is quite dynamic and there quite a lot of textboxes etc...

Thanks!

**Comment by erikringsmuth** [comment]
In short, no it doesn't have two-way binding. This was a concept I was working on back when I was using Backbone.js as my main JS framework. I wasn't a fan of Angular's modules or DI and I wanted to do it better with RequireJS. In the end I started using Ractive and Polymer. Now Angular 2.0 and React are looking like good choices too. I probably won't be doing any further development of Nex.js since there are so many good libraries and frameworks showing up. You could always dig through the source and try adding your own form of two-way binding with object observers or something similar.

**Comment by stephenlautier** [comment]
Great, thanks for you honest answer. Myself was also looking into the RactiveJS which is quite similar to your nex.js with two-way bindings.
I agree, Angular 2.0 along with emerging libs such as Aurelia are really great, however in this case I'm setting a skeleton project which is as simple/easy as possible for someone who doesn't have any knowledge on SPA and not familiar with frameworks such as Angular. Thanks again!
